DepotDownloader/Program.cs

```diff
@@ -11,14 +11,14 @@ namespace DepotDownloader
     class Program
     {
         static void Main( string[] args )
-            => MainAsync( args ).GetAwaiter().GetResult();
+            => System.Environment.Exit( MainAsync( args ).GetAwaiter().GetResult() );

```

Comment:
btw you can make the method signature `int` here and just return the int, there's no need to call `Environment.Exit` from the main function.